src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -440,9 +349,9 @@ public void AnnotationWithoutNonNullTypes_GenericType()
             CSharpCompilation c = CreateCompilation(@"
 public class C<T> where T : class
 {
-    public T? M(T? x1) // warn 1 and 2
+    public T? M(T? x1)
     {
-        T? y1 = x1; // warn 3
+        T? y1 = x1; // warn
         return y1;
     }
 }
```

Comment:
>F(y1, x1) [](start = 8, length = 9)

Warning 3 is a bit surprising. Since the re-inference fails, I'd expect the return type of `F(y1, x1)` to be an error type. Maybe we should not produce that warning?